src/js/utils/UserSettings.js

```diff
@@ -139,5 +141,10 @@
     if (storedGridWidth === 0) {
       ns.UserSettings.writeToLocalStorage_('GRID_WIDTH', 1);
     }
+
+    var storedGridSpacing = ns.UserSettings.readFromLocalStorage_('GRID_SPACING');
+    if (storedGridSpacing === 0) {
+      ns.UserSettings.writeToLocalStorage_('GRID_SPACING', 1);
```

Comment:
You don't need to add that, this was some migration code for version 12, because the GRID_WIDTH setting used to accept the "0" value. I should remove this method now :)